content/en/docs/tutorials/notation-config-file.md

```diff
@@ -0,0 +1,86 @@
+
+### Darwin 
+
+The default location and file will be stored at: `~/Users/exampleuser/Library/Application Support/notation/config.json`.
+
+## Properties
+
+| Property                  | Type     | Value                                                                                                                                                     |
+| ------------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `verificationCerts.certs` | _array_  | collection of name/value pairs for a collection of public certs that are used for verification. These may be replaced with a future policy configuration. |
+| `cert.name`               | _string_ | a named reference to the certificate                                                                                                                      |
+| `cert.path`               | _string_ | a location by which the certificate can be found by the notation cli or notation libraries                                                                |
+| `signingKeys.default`     | _string_ | the signing key to be used when `notation sign` is called without `--name`                                                                                |
+| `signingKeys.keys`        | _array_  | a collection of name/value pairs of signing keys.                                                                                                         |
+| `key.name`                | _string_ | a named reference to the key                                                                                                                              |
+| `key.keyPath`             | _string_ | a location by which the key can be found by the notation cli or notation libraries                                                                        |
+| `key.certPath`            | _string_ | a location by which the paired certificate can be found by the notation cli or notation libraries                                                         |
+| `insecureRegistries`      | _array_  | a list of registries that may be used without https                                                                                                       |
+
+## Samples
+
+See the a sample configuration in a Linux environment: `~/.config/notation/config.json`. 
+
+```json
+{
+    "verificationCerts": {
+        "certs": [
+            {
+                "name": "wabbit-networks",
+                "path": "/home/demo/.config/notation/localkeys/wabbit-networks.crt"
+            },
+            {
+                "name": "import.acme-rockets",
+                "path": "/home/demo/.config/notation/localkeys/import.acme-rockets.crt"
+            }
+        ]
+    },
+    "insecureRegistries": [
+        "registry.wabbit-networks.io"
+    ]
```

Comment:
Also need:
    "credsStore": "pass",
    "credHelpers": {
        "registry.io": "pass"
    },
    "signatureFormat": "jws"

Reply:
What's the `pass` mean here?

Reply:
This is the list of currently available credentials helpers and where you can download them from:

D-Bus Secret Service: https://github.com/docker/docker-credential-helpers/releases
Apple macOS keychain: https://github.com/docker/docker-credential-helpers/releases
Microsoft Windows Credential Manager: https://github.com/docker/docker-credential-helpers/releases
[pass](https://www.passwordstore.org/): https://github.com/docker/docker-credential-helpers/releases